### PR TITLE
Add JustWatch user interview

### PR DESCRIPTION
**Pull request by brian-brazil**

### CODE REVIEW

File: content/blog/2016-10-12-interview-with-justwatch.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+
+## Why did you decide to look at Prometheus?
+
+At JustWatch the public Prometheus announcement hit exactly the right time. We
+mostly had blackbox monitoring for the first few months of the company -
+[CloudWatch](https://aws.amazon.com/cloudwatch/) for some of the most important
+internal metrics, combined with a external services like
+[Pingdom](https://www.pingdom.com/) for detecting site-wide outages. Also, none
+of the classical host-based solutions satisfied us. In a world of containers
+and microservices, host-based tools like Ichinga,
```
Comment (beorn7) [inline content/blog/2016-10-12-interview-with-justwatch.md]: Ichinga → Icinga

File: content/blog/2016-10-12-interview-with-justwatch.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+[Consul-based](https://www.consul.io/) approach. All it takes to get an
+application monitored by Prometheus is to add the client, expose `/metrics` and
+set one simple annotation on the Container/Pod. This low coupling takes out a
+lot of friction between development and operations - a lot of services are
+built well orchestrated from the beginning, because it's simple and fun.
+
+The combination of time-series and clever functions make for awesome alerting
+super-powers. Aggregations that run on the server and treating both
+time-series, combinations of them and even functions on those combinations as
+first-class-citizens makes alerting a breeze - often times after the fact.
```
Comment (beorn7) [inline content/blog/2016-10-12-interview-with-justwatch.md]: first-class citizens

File: content/blog/2016-10-12-interview-with-justwatch.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+For our clients like movie studios or Video on Demand providers, we are an
+international movie marketing company that collects anonymized data about
+purchase behavior and movie taste of fans worldwide from our consumer apps. We
+help studios to advertise their content to the right audience and make digital
+video advertising a lot more efficient in minimizing waste coverage.
+
+![JustWatch logo](/assets/blog/2016-10-12/JW_logo_long_black.jpg)
+
+Since our launch in 2014 we went from zero to one of the largest 20k websites
+internationally without spending a single dollar on marketing - becoming the
```
Comment (beorn7) [inline content/blog/2016-10-12-interview-with-justwatch.md]: Typography nit: The dashes (" - " surrounded by blanks) should be an n-dash or even m-dash ("–" or "—"), depending on your typographical style. There are quite a few in this text.

File: content/blog/2016-10-12-interview-with-justwatch.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+Prometheus enlightened us by making it incredibly easy to reap the benefits
+from whitebox monitoring and label-based canary deployments. The out-of-the-box
+metrics for many Golang aspects (HTTP Handler, Go Runtime) helped us to get to
+a return on investment very quickly - goroutine metrics alone saved the day
+multiple times. The only monitoring component we actually liked before -
+[Grafana](http://grafana.org/) - feels like a natural fit for Prometheus and
+has allowed us to create some very helpful dashboards. We appreciated that
+Prometheus didn't try to reinvent the wheel but rather fit in perfectly with
+the best solution out there. Another huge improvement on predecessors was
+Prometheus' focus on actually getting the math right (percentiles, etc.). In
```
Comment (falschparker82) [inline content/blog/2016-10-12-interview-with-justwatch.md]: Nitpicking: This actually is a highly contested topic in English grammar - http://www.grammarbook.com/punctuation/apostro.asp (Rule 1b&c), if in doubt, both spellings are probably right.

Comment (beorn7) [inline content/blog/2016-10-12-interview-with-justwatch.md]: In doubt, anything is right in English as there is no authoritative source of correctness. I'd try to go with the more common approach, to minimize  confusion. According to my research, "Prometheus's" is the more common way, and your link seems to suggest the same. Plus, it avoids the ambiguity if "Prometheus" is already a plural form.

Comment (falschparker82) [inline content/blog/2016-10-12-interview-with-justwatch.md]: +1, fine with either option

**Comment by brian-brazil** [comment]
Fixed, thanks.

**Pull request merged**